LibertyDSNP/example-client
# Chores/update to sdk 2.0.0 & fix breakages

## Purpose
The big SDK refactor causes a bunch of breakage in example-client if you update to latest. Fix these.
[Fixes #179076239](https://www.pivotaltracker.com/story/show/179076239)

## Solution
1. Upgrade to **SDK 2.0.0** and fix breakages
2. update a couple of packages to fix some vulnerabilities.
3. Get rid of a couple unused packages
4. Update to next nodejs minor version

## To Verify
- All tests should pass
- You should be able to run the example-client, login and create new posts without any errors -- that is, assuming you hold your mouth right, it's a first Tuesday in the month  with a waxing gibbous moon in the 3rd quarter and you've done all the static-server and chain setup correctly.
